conf/application.conf

```diff
@@ -26,7 +26,7 @@ logger.play=INFO
 logger.application=DEBUG
 
 # ReactiveMongo
-mongodb.uri = "mongodb://localhost:27017/scalaplaymongo"
+mongodb.uri = "mongodb://scalaplaymongo:scalaplaymongo123@ds045521.mongolab.com:45521/heroku_app35066918"

```

Comment:
will move it once I figure out how to do environment setup. I need to read another app.config from tests.